doc/quick_start_guide.md

```diff
@@ -156,9 +156,16 @@ Of special interest are:
 
 Apache Flume [web site](http://flume.apache.org/) is also another interesting pointer to learn more about the base technlogy used by Cygnus.
 
-##Contact
-Fermín Galán Marquez (fermin.galanmarquez@telefonica.com)
-<br>
-Francisco Romero Bueno (francisco.romerobueno@telefonica.com)
-<br>
-Germán Toro del Valle (german.torodelvalle@telefonica.com)
+##Reporting issues and contact information
```

Comment:
The same comments apply to this section.

Maybe the "Reporting issues and contact information" section should be in a separate .md to avoid duplication?